### PR TITLE
Fix links for linkchecker build

### PR DESCRIPTION
**Pull request by hflynn**
Should make the linkchecker pass by updating redirecting links and removing broken ones

(also see https://trello.com/c/lvf099f3/234-i3dcore-cbia-3d-library-status)

### CODE REVIEW

**Comment by rleigh-codelibre** [comment]
All looks good to me.  However, I'm getting this from the checker:

```
(line    4) broken    https://cbia.fi.muni.cz/software/ - HTTPSConnectionPool(host='cbia.fi.muni.cz', port=443): Max retries exceeded with url: /software/ (Caused by SSLError(SSLError(1, u'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:726)'),))
(line    4) broken    http://cbia.fi.muni.cz - HTTPSConnectionPool(host='cbia.fi.muni.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, u'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:726)'),))
```

File: sphinx/users/i3dcore/index.rst [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
+`i3dcore <https://cbia.fi.muni.cz/software/>`_,
 also known as the CBIA 3D image representation library, is a 3D image
 processing library developed at the `Centre for Biomedical Image
-Analysis <http://cbia.fi.muni.cz/software-development.html>`_. Together
-with
-`i3dalgo <http://cbia.fi.muni.cz/user_dirs/i3dlib_doc/i3dalgo/index.html>`_
-and `i4dcore <http://cbia.fi.muni.cz/user_dirs/of_doc/libi4d.html>`_,
-i3dcore forms a continuously developed templated cross-platform C++
-suite of libraries for multidimensional image processing and analysis.
+Analysis <http://cbia.fi.muni.cz>`_. Together
```
Comment (rleigh-codelibre) [inline sphinx/users/i3dcore/index.rst]: https ?

File: src/main/resources/format-pages.txt [hunk 2]
```diff
@@ -81,7 +81,7 @@ notes = Known deficiencies: \n
 
 [Amnis FlowSight]
 extensions = .cif
-owner = `Amnis <http://www.amnis.com/>`_
+owner = Amnis (now owned by `Merck <http://www.merckmillipore.com>`_)
```
Comment (rleigh-codelibre) [inline src/main/resources/format-pages.txt]: Also https ?

**Comment by hflynn** [comment]
If those cbia links won't pass the build, I'll have to add them to the ignore list - those ones do at least work manually whereas the previous ones were actual 404s

**Comment by hflynn** [comment]
https://web-proxy.openmicroscopy.org/west-ci/job/BIOFORMATS-build/8/ is green!!

**Comment by jburel** [comment]
note that @sbesson is currently working on BIOFORMATS-docs job peforming the linkcheck

**Pull request merged**